doc/Design.md

```diff
@@ -32,8 +32,22 @@ queue is used.
    - External room: Builds container, fills container, calls `carry` to get
    the energy.
  - `reserver` reservs an external controller and calls `sourcer`.
- - `carry` gets energy from the target container and fills structures and
- storage on the way back. If there is a creep in front the energy is transfered.
+ - `carry`
+    A carry transports energy from the target to the storage in the base (From sources, energy piles or storages in other rooms)
+    They use fixed precalculated paths
+    While moving they try to transfer energy to the next creep or if in base to other structures (tower, link, extension)
```

Comment:
So, that's RCL8 only feature? Why not to transfer into links near sources, if carry pass near them?